### ISSUE TITLE
unexpected DROP TABLE is generated

### ISSUE THREAD

**Issue by KES777** [frioux/DBIx-Class-DeploymentHandler#59]
[this](https://github.com/frioux/DBIx-Class-DeploymentHandler/commit/1813e4691aa746336efe798e973cedec8b4d8ea4) commit introduces the bug 
when for upgrade script DROP TABLE is generated

Now my upgrade scripts look like:
```
--- a/share/migrations/PostgreSQL/upgrade/20-21/001-auto.sql
+++ b/share/migrations/PostgreSQL/upgrade/20-21/001-auto.sql
@@ -4,6 +4,7 @@
 BEGIN;
 
 ;
+DROP TABLE "ips" CASCADE;
 CREATE TABLE "ips" (
   "id" serial NOT NULL,
   "subnet_id" integer NOT NULL,
@@ -16,6 +17,7 @@ CREATE TABLE "ips" (
 CREATE  INDEX "ips_idx_subnet_id" on "ips" ("subnet_id");
 
 ;
+DROP TABLE "localities" CASCADE;
 CREATE TABLE "localities" (
   "id" serial NOT NULL,
   "name" character varying(64) NOT NULL,
```

**Comment by KES777** [comment]
maybe further actions are required because of [TODO](https://metacpan.org/source/ILMARI/SQL-Translator-0.11023/lib/SQL/Translator/Diff.pm#L275)
or default value for `add_drop_table` should be changed to 0 [here](https://metacpan.org/source/MMCCLIMON/DBIx-Class-DeploymentHandler-0.002221/lib/DBIx/Class/DeploymentHandler/DeployMethod/SQL/Translator.pm#L439)

**Comment by frioux** [comment]
Yeah I think I'll remove the default of 1 in your second link. Thanks so
much for tracking the details down here

--

**Comment by frioux** [comment]
Fixed in 0.002222
